### DISCUSSION TITLE
Doubts Regarding FINOS Accessibility Theme Builder Mentorship

### DISCUSSION THREAD

**Discussion by ravjot07** [finos/a11y-theme-builder#860] [General]
Can You guide contributors on what your selection process will look like? Is it based on a project proposal resume and cover letter or some past contributions?
And what would be the timeline for application submissions?

**Comment by aaronreed708** [comment]
Hi @ravjot07, we have reviewed all applications received as of this past Monday.  Having received 115+ applications in this timeframe, we asked the Linux Foundation to close the Mentorship to new applications.  We have been meeting this week to compare notes.  We hope to make a decision early next week.

The kinds of things that we are considering are the information on the cover letter and CV.  We are looking for those candidates with demonstrated experience using the technologies that will be most prevalent in the mentorship project (React, Typescript, Node, Express, etc) as well as experience with frontend development in general.  Speaking for myself, I am also looking at each candidate's leadership positions, previous internships, and experiences on teams in a real-world work environments (such as other open source initiatives).  I have enjoyed looking at deployed code to get a feel for how a candidate codes or designs.  Candidates that are currently making contributions to our project (opening issues, creating documentation, creating pull requests and more) are demonstrating to us their potential by interacting with us in our own environment and showing us their ability to work with our team and that they can learn quickly, which is certainly a plus.  But it is only one of many factors that we are evaluating.  No one has been eliminated from consideration because they have not contributed or because they have not reached out on Slack.

This is our first time hosting a Linux Foundation Mentorship, so we are learning along the way, too.  We hope to offer other mentorships in the future that will, most likely, touch on different disciplines like design, backend development, testing and dev ops.  Theme Builder is certainly a project that holds opportunity for all types of people in tech!

I hope that this answers your question.  If not, please follow up here or reach out on Slack.  As always, thank you for your interest in Theme Builder!

**Comment by ShadabImran07** [comment]
Hi @aaronreed708, I applied for mentorship in this organization but haven't received any kind of acceptance or rejection email yet. I already sent the email with all the details you requested, and my application is still pending. Could you please check on its status?

**Reply by aaronreed708** [reply]
Hi @ShadabImran07, we know that so many people are waiting patiently as we process all of the applications.  We have an update. 
 We did select a candidate and passed that information on to the Linux Foundation this past weekend for processing.  Applicants should be receiving a notification from the LFX Mentorship program this week, I would expect.  It was a very difficult decision.  We had 115+ applicants in just a few weeks, with 65 of these applicants exceeding the minimum requirements.
